examples/observability/assets/podtatohead-deployment-evaluation/post-deployment-tasks.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: lifecycle.keptn.sh/v1alpha1
```

Comment:
I'd suggest to switch to a simple console.log() task, as we now also do in the default podtato head example to make it work out of the box